library.html

```diff
@@ -35,155 +35,174 @@
 			</div>
 			
 			<!--Content Boxes-->
-				<div class = "content_3">
+			<div class = "main_content">
+				
+				<div class = "title2">
 
-					<heading1>
+						<b>
+							<h1>song_id title seconds</h1> <h1>album artist size</h1>
+					   </b>
 						
-						<p>song_id<p>
-						<p>title<p>
-						<p>album<p>
-						<p>genre<p>
-						<p>seconds<p>
-						<p>size<p>
+				</div>
 
-					</heading1>
```

Comment:
The rest of the library page code is kept the same as the previous version apart from the indenting being fixed and the code being moved down to make room for the music display.